Log/lambda_function.py

```diff
@@ -142,8 +142,8 @@ def awslogs_handler(s, event):
     # Get logs
     data = zlib.decompress(base64.b64decode(event["awslogs"]["data"]), 16 + zlib.MAX_WBITS)
     logs = json.loads(str(data))
-    metadata[DD_SOURCE] = "cloudwatch"
-
+    source = logs["logGroup"]
```

Comment:
is `logGroup` always present? Would it be safer to use `logs.get("logGroup", "DEFAULTVALUE")`?

Reply:
It should always be there from cloudwatch logs but will change as you suggest for safety.